docs/learning-the-basics.md

```diff
@@ -102,25 +102,25 @@ The panel, dock, and workspaces appear only on the primary display. Any other at
 
 ## Installing Apps {#installing-apps}
 
-elementary OS comes bundled with **AppCenter**, an app store for free apps. Installing a new app from AppCenter is easy:
+elementary OS comes bundled with **AppCenter**, an app store for apps free of charge. Installing a new app from AppCenter is easy:
```

Comment:
We should probably update this section to cover pay-what-you-want; not all apps are considered free.

Reply:
This probably was meant to mean free as in freedom, not free as in beer. We could change it to Open Source

Reply:
I would use the more precise "libre".